sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/Storage.java

```diff
@@ -68,6 +68,8 @@
 
     private HttpClient mHttpClient;
 
+    private Context mConntext;
```

Comment:
Here's the convention: https://source.android.com/setup/contribute/code-style#follow-field-naming-conventions

```suggestion
    private Context mContext;
```